lib/reek/context/class_context.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'module_context'
+
+module Reek
+  module Context
+    #
+    # A context wrapper for any class found in a syntax tree.
+    #
+    class ClassContext < ModuleContext
```

Comment:
We need an own context for this smell since it should only run on ....classes, not modules. 
Other than that, I believe we should do this anyway and not just alias class processing to module processing.